releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -2246,7 +2246,7 @@
   "124393": {
     "commit": "f39ece24b27062a8d9eb791ffebc252ea1449734",
     "text": "'kubeadm: marked the flag \"--experimental-output'' as deprecated (it will\nbe removed in a future release) and added a new flag ''--output\" that serves the\nsame purpose. Affected commands are - \"kubeadm config images list\", \"kubeadm token\nlist\", \"kubeadm upgrade plan\", \"kubeadm certs check-expiration\".'",
-    "markdown": "'kubeadm: marked the flag \"--experimental-output'' as deprecated (it will\n  be removed in a future release) and added a new flag ''--output\" that serves the\n  same purpose. Affected commands are - \"kubeadm config images list\", \"kubeadm token\n  list\", \"kubeadm upgrade plan\", \"kubeadm certs check-expiration\".' ([#124393](https://github.com/kubernetes/kubernetes/pull/124393), [@carlory](https://github.com/carlory)) [SIG Cluster Lifecycle]",
+    "markdown": "'kubeadm: marked the flag \"--experimental-output'' as deprecated (it will\n  be removed in a future release) and added a new flag ''--output\" that serves the\n  same purpose. Affected commands are - \"kubeadm config images list\", \"kubeadm token\n  list\", \"kubeadm upgrade plan\", \"kubeadm certs check-expiration\".' ([#124393](https://github.com/kubernetes/kubernetes/pull/124393), [@carlory](https://github.com/carlory))",
```

Comment:
```suggestion
    "markdown": "'kubeadm: marked the flag \"--experimental-output'' as deprecated (it will\n  be removed in a future release) and added a new flag ''--output\" that serves the\n  same purpose. Affected commands are - \"kubeadm config images list\", \"kubeadm token\n  list\", \"kubeadm upgrade plan\", \"kubeadm certs check-expiration\".' ([#124393](https://github.com/kubernetes/kubernetes/pull/124393), [@carlory](https://github.com/carlory)) [SIG Cluster Lifecycle]",
```